packages/react/src/components/login/Login.stories.tsx

```diff
@@ -517,6 +594,9 @@ const AuthorizedContent = ({ user }: { user: User }) => {
       <Tabs.TabPanel>
         <NotifyIfErrorOccurs />
       </Tabs.TabPanel>
+      <Tabs.TabPanel>
+        <ProfileData />
```

Comment:
Should this be UserData?

Reply:
No, UserData renders what user data comes from the authorization server (tokens)
ProfileData renders what comes from the Profile Backend.